### PR TITLE
Added 'maxInput' param to WidgetFieldBase (#363)

### PR DESCRIPTION
**Pull request by DRITE**
see #363 

Added availability to set maximum number of characters to fields of type `input` and `text`.

Upgraded antd to 3.26.18 in aims of fixing https://github.com/ant-design/ant-design/issues/21697

### CODE REVIEW

File: src/components/ui/TextArea/TextArea.tsx [hunk 1]
```diff
@@ -8,11 +8,16 @@ import InputDefaultClass from 'antd/lib/input/TextArea'
...
 import ReadOnlyField from '../ReadOnlyField/ReadOnlyField'
 import {TextAreaProps as AntdTextAreaProps} from 'antd/lib/input/TextArea'
+import {WidgetField} from 'interfaces/widget'
 
 type AdditionalAntdTextAreaProps = Partial<Omit<AntdTextAreaProps, 'onChange'>>
 export interface TextAreaProps extends AdditionalAntdTextAreaProps {
+    cursor?:string,
+    widgetName?: string,
+    metaError?: string,
+    meta?: WidgetField, // TODO 2.0.0 must be required?
```
Comment (Dergash) [inline src/components/ui/TextArea/TextArea.tsx:18]: It unnecessary binds a regular UI component to a widget context and narrows its usage, i.e. you'll no longer be able to use this component in other parts of the application.  
UI component shouldn't care about widgets, cursors and meta.

File: src/components/ui/TextArea/TextArea.tsx [hunk 2]
```diff
@@ -8,11 +8,16 @@ import InputDefaultClass from 'antd/lib/input/TextArea'
 import styles from './TextArea.less'
 import ReadOnlyField from '../ReadOnlyField/ReadOnlyField'
 import {TextAreaProps as AntdTextAreaProps} from 'antd/lib/input/TextArea'
+import {WidgetField} from 'interfaces/widget'
```
Comment (Dergash) [inline src/components/ui/TextArea/TextArea.tsx:11]: Should be relative

File: src/interfaces/widget.ts [hunk 3]
```diff
@@ -31,6 +31,10 @@ export const TableLikeWidgetTypes: Array<WidgetTypes | string> = [
     WidgetTypes.PickListPopup
 ]
 
+/**
+ * @param maxInput Maximum number of characters
+ * @param hidden Whether the field is hidden
```
Comment (Dergash) [inline src/interfaces/widget.ts:36]: TSDoc for interface properties should be placed separately for each corresponding property, i.e. `@param` is not valid for interfaces and will not be parsed correctly.

See reference:
https://github.com/microsoft/tsdoc/blob/746a8fb8fe89790107325c79e9077cdc8c83546a/playground/src/CodeEditor.tsx#L5

Comment (DRITE) [reply src/interfaces/widget.ts:36]: I did as you are https://github.com/tesler-platform/tesler-ui/commit/2006015d86bdc0b76af3f9a1f492c877eb30a19e#diff-212d66cacab7e2fade02457e67695c21R132
I'll fix it

**Review by Dergash — APPROVED** [review]

**Pull request merged**